src/App.css

```diff
@@ -1,27 +1,321 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Poppins:ital,wght@0,100;0,600;0,700;1,400;1,500&family=Roboto:ital,wght@0,100;0,600;0,700;1,400;1,500+Slab:ital,wght@0,100;0,600;0,700;1,400;1,500&display=swap");
+
 body {
-  background-color: #dfdfdf;
-  margin: 10%;
+  background-color: #f5f6fa;
+  margin: 4%;
 }
 
 .nav-bar {
   display: flex;
   align-items: center;
   background-color: #fff;
-  padding: 40px;
+  padding: 20px 2%;
   border-radius: 5px;
 }
 
+.brand {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+}
+
 .link {
   padding-left: 20px;
+  opacity: 0.5;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  color: #121212;
+  letter-spacing: 1.9px;
+  text-decoration: none;
+}
+
+.active {
+  opacity: 1;
 }
 
 .icon-button {
   margin-left: auto;
+  color: #0290ff;
+  cursor: pointer;
+  width: 2.813rem;
+  height: 2.813rem;
+  border-radius: 50%;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
 }
 
 .page-container {
-  background-color: #fff;
+  background-color: #fafafa;
   padding: 40px;
-  border-radius: 5px;
+  border-top: 1px solid #d5d5d5;
+}
+
+li {
+  display: list-item;
+}
+
+.books {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 0.938rem;
+}
+
+.book {
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 2rem;
+}
+
+.book-info {
+  display: flex;
```

Comment:
* Kindly dont repeat the same style rule as it is a [CSS bad practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)  as i can see you keep repeating styles like `display flex` `margin` align items centre` etc [Go through this article on how to properly style in CSS](https://speckyboy.com/good-bad-css-practices/)